package.json

```diff
@@ -37,9 +37,10 @@
     "jspdf": "^2.5.1",
     "leaflet": "^1.9.3",
     "lodash": "^4.17.21",
+    "lodash.memoize": "^4.1.2",
     "mathjs": "^11.5.1",
     "node-sass": "^8.0.0",
-    "p5": "^1.5.0",
+    "p5": "^1.3.1",
```

Comment:
Why is the version for p5 downgraded?

Reply:
Check the dependencies of react-p5. While using 1.5.0, they are a problem of type. You can resolve it by using the same dependencies needs by react-p5.

https://github.com/Gherciu/react-p5/blob/main/package.json